src/promotion-builder/features/search-promotions/sub-components/promotion-details/PromotionDetails.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropsTypes from 'prop-types';
+
+const PromotionDetails = ({result}) => {
+
+    const {name, url, devices, ventures} = result;
+
+    return(   
+        <div className="promotionDetails">
```

Comment:
It may be worth thinking about the html here. I think the information should not be represented in a p tag. This feels more like a list to me. Lets catch up offline.